server/models/Notifications.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const ENUM_NOTIFICATIONS = ['Booking', 'Message', 'System', 'Promo'];
+
+const notificationsSchema = new mongoose.Schema({
```

Comment:
Wasn't necessarily explicitly defined in the notification schema but we would actually need to have a reference to who the notification is for. Just the user id

Reply:
I'll include a path that references the `User` schema with `ObjectId` type in the next commit.